modules/snooker/src/main/ts/ephox/snooker/queries/Direction.ts

```diff
@@ -0,0 +1,21 @@
+import { Element as DomElement } from '@ephox/dom-globals';
+import { Fun } from '@ephox/katamari';
+import { Direction, Element } from '@ephox/sugar';
+
+const ltr = {
+  isRtl: Fun.constant(false)
+};
+
+const rtl = {
+  isRtl: Fun.constant(true)
+};
+
+// Get the directionality from the position in the content
+const directionAt = function (element: Element<DomElement>) {
+  const dir = Direction.getDirection(element);
+  return dir === 'rtl' ? rtl : ltr;
+};
+
+export default {
```

Comment:
This doesn't need to be an `export default` from what I can see, so maybe change the above to just be `export const directionAt`?

Also, would this make more sense to be in Sugar instead of here?

Reply:
There's basically a way to do this in Sugar now, so gonna switch it over